InnerEye/ML/configs/histo_configs/classification/DeepSMILEPanda.py

```diff
@@ -76,13 +77,15 @@ def cache_dir(self) -> Path:
 
     def setup(self) -> None:
         if self.encoder_type == InnerEyeSSLEncoder.__name__:
+            from InnerEye.ML.configs.histo_configs.run_ids import innereye_ssl_checkpoint_binary
             self.downloader = CheckpointDownloader(
-                azure_config_json_path=get_workspace(),
-                run_recovery_id=innereye_ssl_checkpoint,
-                checkpoint_filename="last.ckpt",
+                aml_workspace=get_workspace(),
+                run_id=innereye_ssl_checkpoint_binary,  # innereye_ssl_checkpoint
+                checkpoint_filename="best_checkpoint.ckpt",  # "last.ckpt",
                 download_dir="outputs/",
+                remote_checkpoint_dir=Path("outputs/checkpoints")
             )
-            os.chdir(fixed_paths.repository_root_directory())
+            os.chdir(fixed_paths.repository_root_directory().parent)
```

Comment:
[Suggestion] Here and below you could have used the new `repository_parent_directory()`.

Reply:
done